### DISCUSSION TITLE
Change the text of the cookie consent pop-up window

### DISCUSSION THREAD

**Discussion by nurfatimaj** [quarto-dev/quarto-web#92] [Q&A]
Hi!

Is it possible to change the text inside the cookie-consent pop-up window? If I understood correctly, the option `prefs-text` only matters for the link text in the footer of the page.

Thanks!

**Comment by dragonstyle** [comment]
Hello!

It isn't currently possible to customize that text (the underlying js library that we use currently doesn't support this though it is on their roadmap to support soon). I'll take a look and see if there is anything we can do in the meantime...

**Reply by dragonstyle** [reply]
I just took a closer look at what we have to work with here and I think we'll have to wait for the underlying service to add support - there aren't a lot of great ways for us to adapt that text without support from them.

**Reply by nurfatimaj** [reply]
Ok, thanks a lot for the information! Will be waiting for that then.
